lib/puppet/defaults.rb

```diff
@@ -1464,7 +1464,21 @@ def self.valid_file_checksum_types
         with the running configuration.  In the case of puppet master,
         this file reflects the state discovered through interacting
         with clients."
-      },
+    },
+    :statettl => {
+      :default => "7d",
```

Comment:
I'd suggest `32d` to give resources with the longest `monthly` [schedule period](https://puppet.com/docs/puppet/6.0/types/schedule.html#schedule-attribute-period) enough history to function properly. Otherwise, the change in the behavior here will cause existing resources with that schedule to start triggering at odd times.

Reply:
Forgot about the monthly schedule. I was going off of your suggestion of 7 days from the ticket. :) :+1: